spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -21,6 +21,7 @@
 import numpy as np
 from matplotlib.ticker import MaxNLocator
 
+from spinalcordtoolbox.reports import qc2
```

Comment:
Adding a comment here, as I am unable to comment on unchanged lines.

 We should update the `diameter_AP` description below (L81) as it's now measured as average across 3 mm extent

> " - diameter_AP, diameter_RL: Finds the major and minor axes of the cord and measure their length.\n"

Reply:
done here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4958/commits/6442611408586771076b5637916ae5e9d1c33877